docs/src/pages/components/selects/DefaultOpen.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormHelperText from '@mui/material/FormHelperText';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+export default function SelectOtherProps() {
+  const [dog, setDog] = React.useState('');
+  const [cat, setCat] = React.useState('');
+
+  const handleChangeDog = (event: SelectChangeEvent) => {
+    setDog(event.target.value);
+  };
+  const handleChangeCat = (event: SelectChangeEvent) => {
+    setCat(event.target.value);
+  };
+
+  return (
+    <div>
+      <FormControl sx={{ m: 1, minWidth: 120 }}>
+        <InputLabel id="demo-simple-select-dog-label">Dog</InputLabel>
+        <Select
```

Comment:
It seems that you forgot to actually include the `defaultOpen` prop here :)

Reply:
That's actually a matter I wanted to review before merging.

I noticed that when a `Select` component is open, due to the modal style of the menu mount, the page can't be scrolled. Will having the `defaultOpen` prop set to `true` anywhere in the examples included in the docs not be an issue?

I thought maybe a solution would be to have the menu mount inside the example container, but I'm not sure if that's possible. I'm also open to other ideas as to how to best mitigate this.

Reply:
For the time being, I have made those changes to the documentation. Should I also add a warning about this behavior below since I'll be removing the visibility check?

Also, would you have an alternative in mind for mitigating this behavior? It's my personal opinion that it might be detrimental to the developer if they mistakenly set this prop on an invisible component, but it could also be considered a minor inconvenience resulting from not knowing about this.

In any case, I've reverted the visibility check in the code.

Reply:
As this demo is not used, please remove it. The concept of and uncontrolled prop should be simple to understand without a specific demo.